release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -0,0 +1,861 @@
+Authentication and User Agent] for more information.
+
+[[ocp-311-sspi-connection-support-on-windows]]
+==== SSPI Connection Support on Microsoft Windows (Technology Preview)
+
+SSPI Connection Support on Microsoft Windows is currently in
+xref:ocp-311-technology-preview[Technology Preview] and not for production
+workloads.
+
+`oc` now supports the Security Support Provider Interface (SSPI) to allow for SSO
+flows on Windows. If you use the request header identity provider with a
+GSSAPI-enabled proxy to connect an Active Directory server to {product-title},
+users can automatically authenticate to {product-title} by using the `oc`  command
+line interface from a domain-joined Windows computer.
+
+See
+xref:../install_config/configuring_authentication.adoc#windows-sspi-using-request-header[Configuring
+Authentication and User Agent] for more information.
+
+[[ocp-311-notable-technical-changes]]
+== Notable Technical Changes
+
+{product-title} 3.11 introduces the following notable technical changes.
+
+[discrete]
+[[ocp-311-cluster-scoped]]
+==== subjectaccessreviews.authorization.openshift.io and resourceaccessreviews.authorization.openshift.io Are Cluster-scoped Only
+
+*_subjectaccessreviews.authorization.openshift.io_* and
+*_resourceaccessreviews.authorization.openshift.io_* are now cluster-scoped
+*only. If you need namespace-scoped requests, use
+**_localsubjectaccessreviews.authorization.openshift.io_* and
+*_localresourceaccessreviews.authorization.openshift.io_*.
+
+
+[discrete]
+[[ocp-311-scc-new-options]]
+==== New SCC options
+
+SCC has new options:
+
+
+* `AllowPrivilegeEscalation`
+* `DefaultAllowPrivilegeEscalation`
+* `forbiddenSysctls`
+* `allowedUnsafeSysctls`
+
+[discrete]
+[[ocp-311-oc-deploy-removed]]
+==== Removed oc deploy Command
+
+The `oc deploy` command, which was deprecated since {product-title} 3.7, is now
+fully removed. Use `oc rollout` instead.
+
+[discrete]
+[[ocp-311-pipeline-plugin-now-deprecated]]
+====  Pipeline Plug-in Is Deprecated
+
+The {product-title} Pipeline Plug-in is deprecated but continues to work with
+{product-title} versions up to version 3.11. For later versions of
+{product-title}, either use the `oc` binary directly from your Jenkins
+Pipelines, or use the {product-title} Client Plug-in.
+
+[discrete]
+[[ocp-311-logging-es5]]
+====  Logging: Elasticsearch 5
+
+Curator now works with Elasticsearch 5.
+
+See
+xref:../install_config/aggregate_logging.adoc#install-config-aggregate-logging[Aggregating
+Container Logs] for additional information.
+
+[discrete]
+[[ocp-311-oc-env-and-oc-volume-removed]]
+==== Removed oc env and oc volume Commands
+
+The deprecated `oc env` and `oc volume` commands are now removed. Use `oc set
+env` and `oc set volume` instead.
+
+[discrete]
+[[ocp-311-oc-exec-config-patch-command-removed]]
+==== Removed the oc ex config patch Command
+
+The `oc ex config patch` command will be removed in a future release, as it is
+replaced by the `oc patch` command.
```

Comment:
> ==== Removed the oc ex config patch Command

1 Now that it is `in a future release`, the wording should not be Remov**ed** the oc ex config patch Command.
2 I tested in latest v3.11.6, no warning indicates it will be removed in a future release. Where did you get this info?
3
> as it is replaced by the `oc patch` command

`oc patch` does not have the function of `oc ex config patch`. `oc patch` patches openshift resources, `oc ex config patch` patches standalone yaml/json files

Reply:
@xingxingxia https://github.com/openshift/openshift-docs/issues/10458#issuecomment-414808637